udata/core/dataset/views.py

```diff
@@ -103,6 +104,19 @@ def get_context(self):
         return context
 
 
+@blueprint.route('/r/<uuid:id>', endpoint='resource')
```

Comment:
Right now, with the way this is structured, yes (or I need to declare another blueprint).

I agree with the naming pattern (and the fact this outside this PR scope). At the beginning, the URLs were:
* `/d/dataset-slug/ for datasets
* `/o/organization-slug/ for organizations
* `/r/reuses-slug/ for for reuse

There might be something to to, at least for datasets.